packages/odyssey-react-mui/src/Banner.tsx

```diff
@@ -78,9 +79,9 @@ const Banner = ({
       variant="banner"
     >
       <ScreenReaderText>{t(`severity.${severity}`)}:</ScreenReaderText>
-      <AlertTitle>{text}</AlertTitle>
+      <AlertTitle translate={translate}>{text}</AlertTitle>
       {linkUrl && (
-        <Link href={linkUrl} variant="monochrome">
+        <Link href={linkUrl} variant="monochrome" translate={translate}>
```

Comment:
NOTE: `Banner` and other components that contain multiple text elements (`Autocomplete`, `Checkbox`, `Fieldset`, `Form`, `TextField`, `Toast`) will all have to stay in sync in terms of their `translate` value since we can only pass in a single value for `translate` into the component. There are definitely cases where only one of the text elements should be translated but the other should not, but I'm not sure how we should approach this?

Reply:
For `TextField` have you tried [FormHelperTextProps](https://mui.com/material-ui/api/text-field/#TextField-prop-FormHelperTextProps), [InputProps](https://mui.com/material-ui/api/text-field/#TextField-prop-InputProps), [InputLabelProps](https://mui.com/material-ui/api/text-field/#TextField-prop-InputLabelProps) or are they restricted by Odyssey?

Reply:
Looks like those props are **not** available on ODS components